docs/written/builtin/numbers.scrbl

```diff
@@ -855,10 +855,11 @@ decimal.
 
 @examples{
 check:
-  num-to-string-digits(5432.1234, 2) is "5432.12"
-  num-to-string-digits(0.123456789, 2) is "0.12"
```

Comment:
`num-to-string-digits(2/3, 3) is "0.667"` basically shows the same thing. Since I include several more corner case examples, I feel that keeping those two lines will make the documentation too verbose.